model/resource/container.yaml

```diff
@@ -44,7 +47,9 @@ groups:
           K8s defines a link to the container registry repository with digest `"imageID": "registry.azurecr.io
           /namespace/service/dockerfile@sha256:bdeabd40c3a8a492eaf9e8e44d0ebbb84bac7ee25ac0cf8a7159d25f62555625"`.
 
-          OCI defines a digest of manifest.
+          The ID of the same image running in different environments don't not always match.
+          The `oci.manifest.digests` attribute, however, is the same for a given image
+          in all container runtimes that follow OCI specification.
```

Comment:
Was the goal of this sentence to say something like: If you want to have "something" that no matter where it runs will be always the same, use `oci.manifest.digests`? 

If so, then maybe we can tell it in something like this:?

> Consider using `oci.manifest.digests` if it is important to identify the same image in different environments/runtimes